src/main/java/com/epam/pages/popup/AcademicYearsAndVacationPopup.java

```diff
@@ -1,6 +1,4 @@
 package com.epam.pages.popup;
 
-import com.epam.pages.common.CommonPopup;
-
-public class AcademicYearsAndVacationPopup extends CommonPopup {
+public class AcademicYearsAndVacationPopup extends CreatePopup {
```

Comment:
Please extends this class from CommonPopup, becauese you needn't somthing from CreatePopup.

Reply:
done